cns/service/main.go

```diff
@@ -1482,7 +1482,7 @@ func InitializeCRDState(ctx context.Context, httpRestService cns.HTTPService, cn
 		// wait for the Reconciler to run once on a NNC that was made for this Node.
 		// the nncReadyCtx has a timeout of 15 minutes, after which we will consider
 		// this false and the NNC Reconciler stuck/failed, log and retry.
-		nncReadyCtx, cancel := context.WithTimeout(ctx, 15*time.Minute) //nolint // it will time out and not leak
+		nncReadyCtx, cancel := context.WithTimeout(ctx, 15*time.Minute) // nolint // it will time out and not leak
```

Comment:
I believe the nolint pragmas are intended to be directly next to the `//`

```suggestion
		nncReadyCtx, cancel := context.WithTimeout(ctx, 15*time.Minute) //nolint // it will time out and not leak
```

Reply:
here's an arbitrary webpage to say that you're right. wonder why my auto formatter is changing these lines. https://golangci-lint.run/usage/false-positives/#:~:text=tests%20for%20it.-,Use,because,-machine%2Dreadable%20comments

will revert these.

Reply:
except I wonder if `//nolint` by itself does anything and you really need to do `//nolint:all` so my formatter thinks these are just normal comments ... ?

Reply:
`//nolint` by itself probably does nothing... I have always seen it needing a specific linter. Also, there's another linter that complains if you don't have a reason attached.